qiskit/providers/fake_provider/fake_openpulse_2q.py

```diff
@@ -301,6 +301,30 @@ def __init__(self):
                         Nduv(date=mock_time, name="gate_length", unit="ns", value=0.0),
                     ],
                 ),
+                Gate(
+                    gate="u1",
+                    qubits=[1],
+                    parameters=[
+                        Nduv(date=mock_time, name="gate_error", unit="", value=0.06),
+                        Nduv(date=mock_time, name="gate_length", unit="ns", value=0.0),
+                    ],
+                ),
+                Gate(
+                    gate="u2",
+                    qubits=[0],
+                    parameters=[
+                        Nduv(date=mock_time, name="gate_error", unit="", value=0.06),
+                        Nduv(date=mock_time, name="gate_length", unit="ns", value=2 * dt),
+                    ],
+                ),
+                Gate(
+                    gate="u2",
+                    qubits=[1],
+                    parameters=[
+                        Nduv(date=mock_time, name="gate_error", unit="", value=0.06),
+                        Nduv(date=mock_time, name="gate_length", unit="ns", value=2 * dt),
+                    ],
+                ),
```

Comment:
The new logic puts information into the target by `convert_to_target`, but requires a property with sufficient information. So I added for test.